contracts/test/IntegrationTestBase.sol

```diff
@@ -610,13 +612,13 @@ contract IntegrationTestBase is Test, TestParams, TestRegistry, TestSubnetActor,
             })
         );
 
-        saManager = SubnetActorManagerFacet(address(saDiamond));
-        saPauser = SubnetActorPauseFacet(address(saDiamond));
-        saRewarder = SubnetActorRewardFacet(address(saDiamond));
-        saCheckpointer = SubnetActorCheckpointingFacet(address(saDiamond));
-        saGetter = SubnetActorGetterFacet(address(saDiamond));
-        saCutter = DiamondCutFacet(address(saDiamond));
-        saLouper = DiamondLoupeFacet(address(saDiamond));
+        saManager = saDiamond.manager();
+        saPauser = saDiamond.pauser();
+        saRewarder = saDiamond.rewarder();
+        saCheckpointer = saDiamond.checkpointer();
+        saGetter = saDiamond.getter();
+        saCutter = saDiamond.diamondCutter();
+        saLouper = saDiamond.diamondLouper();
```

Comment:
The idea is to get rid of these state fields as they pollute the state, and decompose everywhere we use these.